README.md

```diff
@@ -0,0 +1,345 @@
+
+## Overview 
+
+Kabanero's developer experience for IDEs is provided by Eclipse Codewind.  For more information on how Kabanero and its components work, visit [Kabanero's Architecture and Development Workflows](https://kabanero.io/docs/ref/general/overview/architecture-overview.html).
+
+Kabanero installs on OpenShift and integrates a modern DevOps toolchain and application stack hubs which enable developers to use runtimes and frameworks in pre-built container images called `Application stacks`.  
+
+Eclipse Codewind provides the ability to create application projects for these `Application Stacks` that your company has built, enabling developers to focus on their code and not infrastructure and Kubernetes.  Application deployments to Kubernetes occur via pipelines when developers commit their local code to the correct Git repos Kabanero is managing via webhooks.    
+
+Eclipse Codewind provides the ability to create projects based on a variety of different template types.  These include IBM Cloud starters,Odo, and Appsody templates.  When used with Kabanero, you will only use Appsody templates for the Collections your company has enabled.  Today, there are collections for: Eclipse MicroProfile/Java EE, Springboot, Node.js, Node.js with Express, Node.js with Loopback.
+
+This guide describes developing a simple microservice using two different IDEs. Select the guide section based on your IDE of choice: 
+
+* [Developing with VS Code](#developing-with-vs-code)
+* [Developing with Eclipse](#developing-with-eclipse)
+
+## Developing with VS Code
+
+If you use Visual Studio Code (VS Code), you can use Codewind for VS Code to develop and debug your containerized projects from within VS Code using the workflow you already use today.
+
+### Prerequisite 
+
+Before you can develop a microservice with VS Code, you need to:
+
+* [Install Docker](https://docs.docker.com/install/) 
+    * **Note:** Make sure to install or upgrade to minimum Docker version 19.03. 
+* [Install VS Code](https://code.visualstudio.com/download)
+ 
+### Installing Codewind for VS Code
+
+The Codewind installation includes two parts:
+
+1. The VS Code extension installs when you install Codewind from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) and click `Install`. 
+    * Or, go to `View->Extensions`, search for Codewind, and click `Install`. 
+2. The Codewind backend containers install after you click `Install` when you are prompted. After you click `Install`, the necessary images are downloaded from the internet. The download is approximately 1 GB.
```

Comment:
Clicking "Install" downloads the Codewind backend containers (~1GB).  <-suggestion.